packages/forma-36-react-components/src/components/RadioButton/RadioButton.tsx

```diff
@@ -1,12 +1,13 @@
-import React from 'react';
+import React, { SFC } from 'react';
 import ControlledInput, {
   ControlledInputDefaultProps,
   ControlledInputPropTypes,
 } from '../ControlledInput/ControlledInput';
 
-const RadioButton = props => <ControlledInput {...props} type="radio" />;
+const RadioButton: SFC<ControlledInputPropTypes> = props => (
```

Comment:
I believe that using `SFC` is deprecated. There should `FunctionalComponent` type or something like this.